src/understanding-workflow/components.md

```diff
@@ -94,8 +94,9 @@ configure minio to use persistent storage available in your environment.
 
 The registry component is a managed docker registry which holds application
 images generated from the builder component. Registry persists the Docker image
-iamges to either local storage (in development mode) or to object storage
-configured for the cluster.
+images to either local storage (in development mode) or to object storage
+configured for the cluster. The component also runs a [registry-proxy](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/registry/images) so that the
+registry service is exposed onto each node and available to docker daemon as localhost.
```

Comment:
similar to the above comment (https://github.com/deis/workflow/pull/370/files#r70363312), might want to mention here that 2.1 doesn't do this